### DISCUSSION TITLE
RAMP FOR EXTERNAL FLUX IN PYROSIM

### DISCUSSION THREAD

**Discussion by MarioCruzM** [firemodels/fds#13438] [Q&A]
Hi, I'm working on pyrosim with a liquid fuel model. I'm looking at the example of ethanol on pool fires. The FDS file has the next code to establish a ramp to the external flux to light the liquid fuel:

&SURF ID        = 'POOL'
      COLOR     = 'YELLOW'
      MATL_ID   = 'ETHANOL LIQUID'
      THICKNESS = 0.1
      EXTERNAL_FLUX=20
      **RAMP_EF='flux' /
&RAMP ID='flux', T= 0, F=1 /
&RAMP ID='flux', T=14, F=1 /
&RAMP ID='flux', T=25, F=0 /**

My question is, does anybody know how I can establish this ramp in pyrosim?

**Comment by drjfloyd** [comment]
This forum does not provide support for Pyrosim. Contact Thunderhead for
support.
